Firestore/core/test/firebase/firestore/nanopb/byte_string_test.cc

```diff
@@ -14,65 +14,65 @@
  * limitations under the License.
  */
 
-#include "Firestore/core/src/firebase/firestore/nanopb/nanopb_string.h"
+#include "Firestore/core/src/firebase/firestore/nanopb/byte_string.h"
 
 #include "gtest/gtest.h"
 
 namespace firebase {
 namespace firestore {
 namespace nanopb {
 
-TEST(String, DefaultConstructor) {
-  String str;
+TEST(ByteStringTest, DefaultConstructor) {
+  ByteString str;
   EXPECT_EQ(nullptr, str.data());
 }
 
-TEST(String, FromStdString) {
+TEST(ByteStringTest, FromStdString) {
   std::string original{"foo"};
-  String copy{original};
+  ByteString copy{original};
```

Comment:
std::string isn't implicitly convertible to ByteString, so I don't see how the op== on line 33 could work. (nanopb::String worked because it's op== allowed the second param to be an string_view.)

Reply:
So I'll admit it: I tried to extract this from a much larger change and failed to get it all in here. The test should really be about testing equality of `vector<uint8_t>`. I've pulled those tests forward.

Unfortunately this adds a new method to `ByteString` that I was hoping to leave out of this review: `CopyVector`. I'm not sure I like that name in preference to something simpler like `ToVector`, but I also don't want it to seem as if that vector is cheap. Anyway, PTAL and LMK.

Reply:
OK. Renamed.